spec.go

```diff
@@ -130,7 +130,8 @@ type PostInstall struct {
 // SymlinkTarget specifies the properties of a symlink
 type SymlinkTarget struct {
 	// Path is the path where the symlink should be placed
-	Path string `yaml:"path" json:"path" jsonschema:"required"`
+	Path  string   `yaml:"path" json:"path" jsonschema:"oneof_required=path"`
```

Comment:
We should mark this is deprecated and have people use `paths` instead`.
We can also make it so when we load the spec we migrate `Path` to `Paths` then there's only one field to deal with.

Reply:
How do we mark it as deprecated?

Reply:
https://go.dev/wiki/Deprecated

Reply:
done, but I'm going to add tests for it.

Reply:
tests added.